packages/server/tests/acceptance/rpc_batch1.spec.ts

```diff
@@ -1211,13 +1231,13 @@ describe('@api-batch-1 RPC Server Acceptance Tests', function () {
         const transaction = {
           type: 2,
           chainId: Number(CHAIN_ID),
-          nonce: await relay.getAccountNonce(accounts[2].address, requestId),
+          nonce: await relay.getAccountNonce(accounts[1].address, requestId),
```

Comment:
note for reviewer: since accounts[2] is the payers of most of the test, when it reaches this case, system throws Insufficient Fund error. Therefore, this change is made to update the payer to account[1] for this case.